src/Symfony/Bridge/Twig/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+4.4.0
+-----
+
+ * deprecated to pass `$rootDir` and `$fileLinkFormatter` as 5th and 6th argument respectively, swaps the variables position.
```

Comment:
it's missing a reference to the relevant class

Reply:
Fixed.